app/models/profile.rb

```diff
@@ -3,12 +3,47 @@ class Profile < ApplicationRecord
 
   validates :user_id, uniqueness: true
 
-  # This method generates typed accessors for all profile fields
-  def self.refresh_store_accessors!
+  USER_MIXIN = Module.new
```

Comment:
We're building this mixin up dynamically, see https://dev.to/appsignal/configurable-ruby-modules-the-module-builder-pattern-4483 for more info. I'll add a comment later.